content/tutorials/filesystem-keystore.md

```diff
@@ -99,25 +99,25 @@ Remember to adjust the `tls` config section.
 
 4. Start KES Server
    
-   ```
-   $ kes server --config config.yml --auth off
+   ```sh  {.copy}
+   kes server --config config.yml --auth off
    ```
    
    {{< admonition title="Linux Swap Protection" type="tip" >}}
 
    In Linux environments, KES can use the [`mlock`](http://man7.org/linux/man-pages/man2/mlock.2.html) syscall to prevent the OS from writing in-memory data to disk (swapping). 
```

Comment:
oh hey here's a link for `mlock` if you want to include it in the other mentions